test/ace/http1/server_test.exs

```diff
@@ -632,4 +632,33 @@ defmodule Ace.HTTP1.ServerTest do
   end
 
   # DEBT try sending empty fragment with end_stream false? should not end stream
+
+  test "If a worker dies while sending a chunked response, its endpoint dies gracefully", %{
+    port: port
+  } do
+    http1_request = """
+    GET / HTTP/1.1
+    host: example.com
+
+    """
+
+    {:ok, socket} = :ssl.connect({127, 0, 0, 1}, port, [:binary])
+    :ok = :ssl.send(socket, http1_request)
+
+    assert_receive {:"$gen_call", from = {worker, _ref}, {:headers, _request, _state}}, 1000
+
+    response =
+      Raxx.response(:ok)
+      |> Raxx.set_body(true)
+
+    GenServer.reply(from, response)
+
+    {Raxx.Forwarder, _forwarder_state, {:http1, endpoint, _}} = :sys.get_state(worker)
+    endpoint_monitor = Process.monitor(endpoint)
+
+    send(endpoint, {:DOWN, :ref, :process, worker, :normal})
```

Comment:
Instead of this line we could also do `Process.exit(worker, :foo)` and then `assert endpoint_exit_reason == :foo`

Reply:
I assume using `Process.exit` would add another unwanted bit of error logging when running the tests? In which case this'll do. The other alternative would be to have Raxx.Forwarder understand a message that causes it to exit. something like `handle_info({Raxx.Forwarder, :stop, :normal}, state)`

Reply:
Yeah, there's a good amount of error logging in the other case that doesn't look pretty and is difficult to `capture_io()`.

I like the Raxx.Forwarder idea, I'll do that.